docs/tests.md

```diff
@@ -1,11 +1,11 @@
 ## Test
 
-- Run local tests against source code. </br> `poe test_src` </br> </br> 
+- Run local tests against source code. </br> `poe test` </br> </br> 
```

Comment:
We should say which is the default profile

Reply:
There are no more profiles, that's why I opened #162